packages/code/src/Syntax.tsx

```diff
@@ -98,14 +96,14 @@ function Syntax({
       highlightedContent.react
     );
 
-  const mode = darkMode ? Mode.Dark : Mode.Light;
+  const { theme, darkMode } = useCodeContext();
 
   const baseFontSize = useBaseFontSize();
-  // TODO: Refresh - remove darkMode logic
-  const codeFontSize = baseFontSize === 14 ? 13 : darkMode ? 16 : 15;
-
+  // TODO: remove 14 check when useBaseFontSize is updated
+  const codeSize = baseFontSize === 14 ? '1' : '2';
   const codeFontStyles = css`
-    font-size: ${codeFontSize}px;
+    font-size: ${typeScales[`code${codeSize}`].fontSize}px;
+    line-height: ${typeScales[`code${codeSize}`].lineHeight}px;
```

Comment:
This is a little confusing. I think using the ternary to select the typescale would be a little clearer.
```js
const typeScale = (baseFontSize === 14) ? typeScales['code1'] : typeScales['code2']
```

Reply:
(we should have a utility that converts `baseFontSize` to a relevant `typeScale`